WordPress/Classes/Networking/LoginClient.swift

```diff
@@ -3,7 +3,7 @@ import WordPressAPI
 import AutomatticTracks
 
 actor LoginClient {
-    
+
```

Comment:
SwiftLint picked this and the ones below in CI, so I addressed them because I like the look of a green build 🤓